### PR TITLE
Update de.csv

### PR DESCRIPTION
**Pull request by billy995**
translation mistake on new-employee-checkin
IN,IM to IN,EIN

### CODE REVIEW

File: erpnext/translations/de.csv [hunk 1]
```diff
@@ -6003,7 +6003,7 @@ CN,CN,
 DE,DE,
 ES,ES,
 FR,FR,
-IN,IM,
+IN,Ein,
```
Comment (marination) [inline erpnext/translations/de.csv:6007]: @billy995  hey can you give some instances where **Ein** is used as IN (context IN/OUT) ? From everything I can find online **IN** translates to IN (english) and **IM** translated to IN THE (english)

EIN seems to translate to "a/an" (article)

**Comment by billy995** [comment]
https://github.com/frappe/erpnext/issues/31536#issue-1294820239

Marica ***@***.***> schrieb am Di., 19. Juli 2022, 11:03:

File: erpnext/translations/de.csv (hunk 1 above)
Comment (billy995) [reply erpnext/translations/de.csv:6007]: @marination  on new-employee-checkin [example : stamp in = einstempeln   stamp out = ausstempeln] in = ein, out = aus
grafik

Comment (marination) [reply erpnext/translations/de.csv:6007]: Had to check if IN was used elsewhere in other context. Nothing so far so this is good to go

**Review by marination — APPROVED** [review]

**Pull request merged**